flavors/netmesh/netmesh_flavor.go

```diff
@@ -86,6 +88,9 @@ func (f *FlavorNetmesh) Inject() (allReadyInjected bool) {
 	f.Netmesh.StatusMonitor = &f.StatusCheck // StatusCheck included in local.FlavorLocal
 	f.CRD.Deps.PluginInfraDeps = *f.FlavorLocal.InfraDeps("netmeshcrd")
 	f.CRD.Deps.KubeConfig = config.ForPlugin("kube", KubeConfigAdmin, KubeConfigUsage)
+	f.Handler.Deps.PluginInfraDeps = *f.FlavorLocal.InfraDeps("netmeshhandler")
+	f.Handler.Deps.KubeConfig = config.ForPlugin("kube", KubeConfigAdmin, KubeConfigUsage)
+	f.CRD.HandlerAPI = &f.Handler
```

Comment:
nit: please move line 93 under line 90.

Reply:
Done

Reply:
I meant to group f.CRD together, so 89, 90 and 93 are together.